modules/fortomte/fortomte.nf

```diff
@@ -118,7 +118,7 @@ process ESTIMATE_HB_PERC {
     path hb_genes
 
     output:
-    path ("${meta.sample}_perc_mapping.json"), emit: tsv
+    tuple val(meta), path("${meta.sample}_perc_mapping.json"), emit: tsv
```

Comment:
`ESTIMATE_HB_PERC` is executed as part of Tomte. So I don't think we should need using it here.